### DISCUSSION TITLE
How to evaluate the vector crosss product?

### DISCUSSION THREAD

**Discussion by xwaeaewcrhomesysplug** [axkr/symja_android_library#249] [Q&A] [answered]
**Describe the bug**
vector crosss product.seems to be wrong consistently.

eg,a={5,1,4} ,b={-1,0,2}

maths solution process(idk what is this technique name)
axb
=
|i j k|
|5 1 4|
|-1 0 2|

=[(1*2)-(4*0)]i-[(5*2)-(4*-1)]j+[(5*0)-(-1*1)]k

=2i-14j+k

<2,-14,1>

**To Reproduce**

Example input Symja input expression:

```
>> {5,1,4}*{-1,0,2}
={-5,0,8}

```

Error result:{-5,0,8}

```

```

**Expected behavior**

```
Expected (correct) result expression:{2,-14,1}

```

==

**Java**
 - Java15
-Win10

**Comment by axkr** [comment, marked as answer]
Please use the `Cross` function for cross product:

```mma
>> Cross({5,1,4},{-1,0,2})
{2,-14,1}
```

The `Times` operator `*` multiplies element-wise.

See:
- https://github.com/axkr/symja_android_library/blob/master/symja_android_library/doc/functions/Cross.md
